### ISSUE TITLE
Wrong Baseline Result

### ISSUE THREAD

**Issue by Elizalll** [Puzzled-Hui/ATM-22-Related-Work#6]
Dear organizers @Puzzled-Hui @yungu-imr :

I am writing to ask for some solutions for my wrong baseline result.

I download `predict.py`, `model.py` and `weight.pth` from https://github.com/EndoluminalSurgicalVision-IMR/ATM-22-Related-Work/tree/main/baseline-and-docker-example.

I run it in a conda environment, there is no error message and the result is saved. However, I found the inference result is abnormal. It didn't predict the airway correctly. Instead, it predicts an organ underneath the lungs for validation set and some noises(maybe) for training set. (Screenshot from ITK-SNAP is attached)

In the file I only modified the weight path and the input and output path. The normal saving of the file means that there is no problem with my input and output paths. I tried removing the line of code that loads the weights, and the result was changed to predict true for almost all regions, which indicates that the weights file was also loaded correctly.

**Is there an error in the weight file or code? Could you please tell me what I can do to get the correct baseline result?**

Result example: ATM_440_0000 from **validation set**:
<img width="736" alt="image" src="https://github.com/user-attachments/assets/43d633af-388e-4abb-b692-f56bd34c51ea">

Result example: ATM_001_0000 from **training se**t:
<img width="693" alt="image" src="https://github.com/user-attachments/assets/e493dfcb-55df-43d6-90a6-8595b5c70cce">

My conda environment: 
python==3.8 
pytorch==1.7.0 
torchvision==0.8.0 
torchaudio==0.7.0 
cudatoolkit=11.0 
numpy==1.19.5 
monai==0.8.1
SimpleITK==2.0.2
h5py==3.1.0
nibabel==3.2.1
scikit-image==0.17.2
tqdm==4.54.1
pandas==1.1.5

Yours sincerely,
Eliza Lin

**Comment by Puzzled-Hui** [comment]
Hi, 

Thanks for your interest, we will test this baseline and reply to you. Thank you for your patience :)

**Comment by Puzzled-Hui** [comment]
Hi,

Thanks for your patience, we have tested the baseline result in this repo however we did not encounter your reported errors.  

Seen below:
![image](https://github.com/user-attachments/assets/af4ccfba-7cd8-4a80-85f7-159cfa4a7bbe)
